docs/concepts/sensor-models/sensor-models.md

```diff
@@ -0,0 +1,88 @@
+
+### Integrated Software for Imagery and Spectrometry (ISIS)
+[ISIS](https://github.com/DOI-USGS/ISIS3) is a software suite that comprises a variety of image analysis tools ranging from traditional tools like contrast, stretch, image algebra, and statistical analysis tools to more complex processes such as mosaicking, photometric modeling, noise reduction, and bundle adjustment. In addition to these analysis tools, ISIS defines the .cub format which can capture 3-dimensional image data (lines, samples, and bands) as well as image metadata in the form of an image label.  In addition to capturing image data and metadata, .cub files can be augmented with camera geometry information such that elements of the camera model are captured alongside the data.
+
+#### Bundle Adjustment / Jigsaw
+ISIS's Jigsaw program provides a bundle adjustment algorithm, which is a full 3-dimensional scene reconstruction algorithm that
+
+1. provides estimates of the 3-dimensional coordinates of ground points
+1. provides estimates of the sensor's exterior orientations
+1. minimizes error between the reconstructed scene and observed point locations
+
+Bundle adjustmnet is a critical part of the sensor model ecosystem in that it can be used to iteratively refine and correct a model's geometry, which allows for more accurate transitions between reference frames, i.e. correctly geolocating a ground point from an image plane.
+
+More information related to bundle adjustment can be found [here](https://isis.astrogeology.usgs.gov/Application/presentation/Tabbed/jigsaw/jigsaw.html)
+
+#### ISIS Camera Models
+Because ISIS predates the CSM, it contains camera models that are not compliant with the CSM API. ISIS contains camera models for framing, pushframe, linescan, radar, point, and rolling shutter cameras. While ISIS's cameras models are authoritative and mathematically correct, they are only usable within the context of ISIS.  However, ISIS has also been modified to interoperate with CSM cameras.  While ISIS camera models are still actively used, efforts are being taken to replace these proprietary models with CSM compliant models via the USGSCSM library.
+
+### USGS Community Sensor Model (USGSCSM)
+The [USGS Community Sensor Model](https://github.com/DOI-USGS/usgscsm) is a concrete implementation of sensor models according to the standards described in the CSM.  Where the CSM is the set of rules and standards that guides the creation of sensor models, the USGSCSM is a library of sensor models that adheres to those rules.  The USGSCSM library provides generic instances of sensor models for instantaneous (framing) cameras, time-dependent (line-scan) cameras, push-frame cameras, synthetic aperture radar (SAR), and projected sensor models.  Additionally, USGSCSM provides an extensible plugin architecture that allows for additional, interface-compliant sensor models to be dynamically loaded.
```

Comment:
Consider defining what a projected sensor model is. If the data are projected and the sensor model has a way to undo the projection, then the sensor model isn't really projected, it just has capability to handle data which are. 

I know it is pedantic, but the topic is really complex and I fear we might hear about projected sensor models at a conference.

Reply:
Hmm. I think I misunderstood what projected sensor models were regarding "if the sensor model has a way to undo the projection, then it's not really projected."  I can't really find good info on the particulars of projected sensor models.  Do you have a resource in mind that I can check out?

Reply:
I do not. From my perspective a projected sensor model is not a thing. One has a sensor model. One can then apply a map projection to an observation. If software exists to take a sensor model and a map projected image and get back information as defined by the CSM API then that is an implementation detail and not a whole new thing.

Reply:
@AustinSanders Maybe a way to reword it is "sensor model support for projected images" rather than a projected sensor model. The projection is undone then the camera model is used. So, it's like an extra layer of distortion, so to speak, on top of the camera model. But the camera model itself cannot be considered projected.